bot.py

```diff
@@ -259,15 +259,15 @@ def registerTexts(self):
 
 /listall - Lists all possible groups to watch.
 
-/noplus1 - Enables +1 filtering.<b>Experimental</b>.
+/noplus1 - Enables +1 filtering.*Experimental*.
```

Comment:
could you move this to markdown commit ?

Reply:
Done